src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -546,7 +546,7 @@ public static LogisticRegressionBinaryTrainer LogisticRegression(this BinaryClas
         }
 
         /// <summary>
-        /// Predict a target using a linear regression model trained with the <see cref="Microsoft.ML.Trainers.PoissonRegressionTrainer"/> trainer.
+        /// Predict a target using a linear regression model trained with the <see cref="Microsoft.ML.Trainers.PoissonRegressionTrainer"/>.
         /// </summary>
```

Comment:
></summary> [](start = 12, length = 10)

Can you please add a remarks section with a discussion of what Poisson regression is? We've had users who are trying to use this for plain linear regression, and this *is* a bit specialized.

Reply:
Oh, nevermind -- I see you did this is the base trainer.

---
In reply to: [268295289](https://github.com/dotnet/machinelearning/pull/3059#discussion_r268295289) [](ancestors = 268295289)